### DISCUSSION TITLE
Print nested

### DISCUSSION THREAD

**Discussion by yajo** [outline/outline#3047] [Ideas]
Imagine I want to handle a PDF to somebody who needs offline access to docs and doesn't understand markdown.

It'd be great to be able to print a document with nested documents included.

**Comment by tommoor** [comment]
This is a good idea, it has been raised before. I’m not sure if this is
possible using the system print dialogue and will probably require custom
PDF generation

**Reply by almereyda** [reply]
Maybe it could work through an intermediate page, which loads and displays a tree of documents into one, and then prints it?

**Comment by michaelkarrer81** [comment]
This one would be really great! Enables us to create manuals with outline!
Should be merged with https://github.com/outline/outline/discussions/2644
